src/transformers/modeling_roberta.py

```diff
@@ -57,15 +65,31 @@
 ]
 
 
-class RobertaEmbeddings(BertEmbeddings):
+RobertaLayerNorm = torch.nn.LayerNorm
+
+
+class RobertaEmbeddings(nn.Module):
     """
     Same as BertEmbeddings with a tiny tweak for positional embeddings indexing.
     """
 
+    # Copied from transformers.modeling_bert.BertEmbeddings.__init__ with Bert->Roberta
     def __init__(self, config):
-        super().__init__(config)
+        super().__init__()
+        self.word_embeddings = nn.Embedding(config.vocab_size, config.hidden_size, padding_idx=config.pad_token_id)
+        self.position_embeddings = nn.Embedding(config.max_position_embeddings, config.hidden_size)
+        self.token_type_embeddings = nn.Embedding(config.type_vocab_size, config.hidden_size)
+
+        # self.LayerNorm is not snake-cased to stick with TensorFlow model variable name and be able to load
+        # any TensorFlow checkpoint file
+        self.LayerNorm = RobertaLayerNorm(config.hidden_size, eps=config.layer_norm_eps)
+        self.dropout = nn.Dropout(config.hidden_dropout_prob)
+
+        # position_ids (1, len position emb) is contiguous in memory and exported when serialized
+        self.register_buffer("position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)))
+
+        # End copy
```

Comment:
This special comment tells the script to stop looking at the code at this point, allowing us to have custom stuff after. This is the equivalent of doing
```
super().__init__(config)
subclass_specific_stuff
```